### PR TITLE
WIP: IE css fixes

### LINKED ISSUE #54: Test all pages in ie
Check the web site on ie 9 and 10

### PR DESCRIPTION
**Pull request by bitamar**
#54

### CODE REVIEW

**Comment by DavidGizra** [comment]
@bitamar - you can check and close.
Thx.

File: jekyll/app/_scss/ie.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+body {
+  .page-books .wrapper .books-list .letter-wrapper {
+    clear: both/9;
```
Comment (bitamar) [inline jekyll/app/_scss/ie.scss]: Since we load this css only on IE <= 9, theres no need for the "/9" hack (Or is there a reason?).

Comment (bitamar) [inline jekyll/app/_scss/ie.scss]: Please remove the /9 also from here, and verify the fixes still work for ie 9.

**Comment by DavidKohav** [comment]
IE 9 pixel perfect home page.
homepage

**Comment by DavidKohav** [comment]
IE 10-11 pixel perfect home page.
homepageie10

**Comment by DavidKohav** [comment]
IE 10-11 pixel perfect home page.
dsdsds

**Comment by bitamar** [comment]
Merged

**Comment by DavidKohav** [comment]
Add a book ie 10-11
bookie

**Comment by DavidKohav** [comment]
Add a book ie 10-11
book-bottom

**Comment by DavidKohav** [comment]
books page on ie-9:
booksie9

**Comment by DavidKohav** [comment]
books in ie 10-11:
books ie 10-11

**Comment by DavidKohav** [comment]
@bitamar - After you will see the green V you can merge the issue to the master :)

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**